src/main/kotlin/no/nav/syfo/aktivitetskrav/api/AktivitetskravApi.kt

```diff
@@ -52,6 +52,15 @@ fun Route.registerAktivitetskravApi(
 
             call.respond(responseDTOList)
         }
+        post {
+            val personIdent = call.personIdent()
+            //val requestDTO: NewAktivitetskravDTO? = call.receiveNullable<NewAktivitetskravDTO>()
```

Comment:
Er det ikke litt voldosomt å bruke et helt objekt og body for dette? Hvorfor ikke en header eller queryparam? Evt. hvorfor kan ikke bare backend selv finne ut om det finnes aktivitetskrav i dette tilfellet eller ikke?

Reply:
Det ryddigste hadde kanskje vært å hatt to endepunkter ala:
`POST /aktivitetskrav` for å lage et nytt aktivitetskrav når det ikke finnes noe fra før
`POST /aktivitetskrav/{uuid}/ny-vurdering` (e.l. navn) for å lage nytt når aktivitetskrav med gitt `uuid` er i sluttilstand

Kan vurdere det hvis det blir vanskelig med nullable body.

Reply:
Http headers er ofte brukt for forskjellige metadata for nettverkskall. Lurer på om det blir rart å legge noe som egentlig er en instruks til hvordan den nye aktivitetskrav ressursen skal lages her.
Query params også har en litt annet bruksområde slik jeg har forstått det, så føles feil å legge en payload her. Dette er oftest brukt til filtering i GET requester.

Reply:
Jeg tenker at dette ikke trenger å være en payload da. Dette _kan_ i hvert fall tolkes som en param som "filtrerer" hvilket aktivitetskrav som skal få ny vurdering. (Men da er kanskje path param det greieste som Anders nevner.)